testing/ncids-css-testing/stories/components/nci-autocomplete/nci-autocomplete.scss

```diff
@@ -0,0 +1,129 @@
+@import '@nciocpl/ncids-css/scss/lib/normalize';
+@import '@nciocpl/ncids-css/scss/packages/required';
+@import '@nciocpl/ncids-css/scss/packages/global';
+@import '@nciocpl/ncids-css/scss/components/nci-header/nci-extended';
+@import 'uswds/dist/scss/base/accessibility';
```

Comment:
Should this be in global or something?

Reply:
yeah, I think it should be now.  making this change